libraries/Matter/keywords.txt

```diff
@@ -24,6 +24,19 @@ MatterContactSensor	KEYWORD1
 MatterPressureSensor	KEYWORD1
 MatterOccupancySensor	KEYWORD1
 MatterOnOffPlugin	KEYWORD1
+MatterThermostat	KEYWORD1
+ControlSequenceOfOperation_t	KEYWORD1
+ThermostatMode_t	KEYWORD1
+EndPointCB	KEYWORD1
+EndPointHeatingSetpointCB	KEYWORD1
+EndPointCoolingSetpointCB	KEYWORD1
```

Comment:
Can `EndPoint` be replaced with `Thermostat` on those?

Reply:
EndPointCB is used in all the endpoint with the same name, but using its own qualifier.
Example: MatterThermostat::EndPointCB<bool(uint8_t)> 
This is name given to the type of the generic attribute callback.